### DISCUSSION TITLE
Redis 7.x cluster support

### DISCUSSION THREAD

**Discussion by jaysonhurd** [redis/rueidis#498] [Q&A]
Does Rueidis 1.0.31 support Redis 7.x cluster?  We are using Centrifugo, need to upgrade our Redis cluster to 7.x from 6.x and beforehand would like to know if I also update Centrifugo (which will use Rueidis 1.0.31) if 7.x cluster is supported.  Thanks!!

**Comment by rueian** [comment]
Hi @jaysonhurd, yes, rueidis can work with redis 7.x cluster.
